onnxruntime/core/common/threadpool.cc

```diff
@@ -22,6 +22,28 @@
 
 using Eigen::Barrier;
 
+namespace {
+//Copied from MlasPartitionWork
```

Comment:
>/Copied from MlasPartitionWork [](start = 1, length = 30)

I guess this more evenly distributes the work that is extra. Previously, it was the last thread that was picking up the slack?

Reply:
Right. I'll put comments there, in a later PR.